web_generator/mono_pkg.yaml

```diff
@@ -9,8 +9,12 @@ stages:
   - test: --test-randomize-ordering-seed=random -p chrome
   - test: --test-randomize-ordering-seed=random -p chrome -c dart2wasm
 - generate_and_analyze:
-  - command: dart bin/update_bindings.dart
-  - analyze: --fatal-infos .
+  - command:
+    - dart pub -C ../web get
+    - dart bin/update_bindings.dart
+    - dart analyze --fatal-infos ../web
```

Comment:
before we were analyzing `web_generator`! We want to analyze `web` after `update_bindings` right?

@srujzs

Reply:
Each repo has a `mono_pkg.yaml` file. Here, we're analyzing the `web_generator`, but there are separate jobs to analyze `web` in the `mono_pkg.yaml` file under `web`.

Reply:
I know that! 😛 But we want to validate `web` after we run the `update_bindings` verification, which makes sense to have under `web_generator`

Reply:
Oh! Yes, it makes no sense to analyze `web_generator` after the generation. I added this specifically to get info on whether new IDL versions break us, so this LGTM.